addons/main/functions/fnc_drawDownedUnitIndicator.sqf

```diff
@@ -14,4 +14,13 @@ private _color = [
     if (_distance > GVAR(showDownedUnitIndicatorRange)) then {continue};
     private _sizeFinal = linearConversion [0, GVAR(showDownedUnitIndicatorRange), _distance, _size, _size/10, false];
     drawIcon3D [["\a3\ui_f\data\igui\cfg\actions\bandage_ca.paa", "\A3\ui_f\data\Map\VehicleIcons\pictureHeal_ca.paa"] select (_x getVariable [QGVAR(beingRevived), false]), _color, (ASLtoAGL visiblePositionASL _x) vectorAdd [0, 0, 0.5], _sizeFinal, _sizeFinal, 0, "", 0, 0, "RobotoCondensed", "center", true, 0, 0.5];
+    if (!isNull player) then {
+        if (_distance > GVAR(bleedoutTimerRange)) then {continue};
+        private _timeRemaining = _x getVariable [QGVAR(bleedoutTimeLeft), -1];
+        private _canseeTimer = _timeRemaining >= 0 && GVAR(visibleBleedoutTimer) > 0 && (player getUnitTrait "Medic" || GVAR(visibleBleedoutTimer) == 2);
```

Comment:
Use "lazy evaluation" in sqf here (https://community.bistudio.com/wiki/Code_Optimisation#Lazy_evaluation)

offload `player getUnitTrait "Medic"` outside the loop, as it does not need to be queried for every downed unit in range.